src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -165,61 +167,63 @@
 
               <!-- Roles -->
               <template v-if="isCorrectionFiling">
-                <label class="sub-header">Roles</label>
-                <v-row class="roles-row my-6">
-                  <v-col cols="4" class="mt-0" v-if="isPerson">
-                    <div class="pa-1">
-                      <v-checkbox
-                        id="cp-checkbox"
-                        class="mt-1"
-                        v-model="selectedRoles"
-                        :value="RoleTypes.COMPLETING_PARTY"
-                        :label="RoleTypes.COMPLETING_PARTY"
-                        :rules="roleRules"
-                        @change="assignCompletingPartyRole()"
+                <article class="mt-6">
```

Comment:
If you go to the ⚙️ icon at the top of the files pages and hide whitespace, it's easier to see what I changed here (ie, not very much).